vncdotool/rfb.py

```diff
@@ -209,6 +210,8 @@ def _handleInitial(self):
                             % version_server)
                     version = max(filter(
                         lambda x: x <= version_server, SUPPORTED_VERSIONS))
+            else:
+                log.msg("Server did not return expected version format: %s", buffer[:30])
```

Comment:
This should raise an exception rather than silently fail, and continue processing as if there is a valid connection